op-e2e/actions/proofs/operator_fee_test.go

```diff
@@ -134,7 +143,7 @@ func Test_ProgramAction_OperatorFeeConsistency(gt *testing.T) {
 			// Send an L2 tx
 			env.Sequencer.ActL2StartBlock(t)
 			env.Alice.L2.ActResetTxOpts(t)
-			env.Alice.L2.ActSetTxToAddr(&env.Dp.Addresses.Bob)
+			env.Alice.L2.ActSetTxToAddr(&env.Dp.Addresses.Bob)(t)
```

Comment:
In this case it didn't really impact the test logic, but this "function returning a function" pattern for the action API is super footgun ridden. gopls LSP doesn't pick it up as a potential issue, and it's caused me some hard to debug headaches before. Really would like to see this API get refactored.

Reply:
agree... as a first step we should just reduce all `Act` functions to normal calls.

Reply:
tracking https://github.com/ethereum-optimism/optimism/issues/15069